### PR TITLE
chore: cache dependencies

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

**Comment by clue** [comment]
Thank you for working on this PR, perhaps you can elaborate on what's the motivation for this change? :+1:

**Comment by WyriHaximus** [comment]
> Thank you for working on this PR, perhaps you can elaborate on what's the motivation for this change? 👍

For my own packages I'm doing this to speed up the build time.

**Comment by clue** [comment]
> > Thank you for working on this PR, perhaps you can elaborate on what's the motivation for this change? +1
>
> For my own packages I'm doing this to speed up the build time.

I understand the *potential* speed improvements, but has build time really ever been an issue in this project? I would rather KISS :man_shrugging:

**Comment by DanielRuf** [comment]
> I understand the potential speed improvements, but has build time really ever been an issue in this project? I would rather KISS 🤷‍♂️

If the performance of the builds and less downloads of packages (= not so often breaking npm install) don't matter then I would say we can close this PR as this is just a build cache to speed things up and have some offline cache of dependencies.

File: .travis.yml [hunk 1]
```diff
@@ -15,3 +15,7 @@ install:
 
 script:
   - php vendor/bin/phpunit --coverage-text
+
+cache:
+  directories:
+    - "$HOME/.composer/cache/files"
```
Comment (peter279k) [inline .travis.yml]: The Travis setting missed the EOF(end of file).

**Comment by peter279k** [comment]
The changed `.travis.yml` has conflicts and it should be fixed.

**Comment by DanielRuf** [comment]
I guess you mean "please rebase".

I'll do so later today.

**Comment by peter279k** [comment]
@DanielRuf, that's correct. Thank you for your concern.

**Comment by clue** [comment]
@DanielRuf Thank you for your contribution! I can see where you're coming from, but in this instance I'm leaning towards *if it ain't broke don't fix it*.

I'm closing this for now as it hasn't received any input in a while and I believe this has been answered. Please come back with more details if this problem persists and we can reopen this :+1:

I hope you don't see this as discouraging, so again thank you for your effort and keep it coming!

**Pull request closed without merging**